libexec/integration-test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file This script starts dev server and runs cypress tests.
+ */
+
+const { spawn } = require('child_process')
+spawn('npx', ['http-server', 'dist'],{shell:true})
+const waitOnProcess = spawn('npx', ['wait-on', 'http-get://localhost:8080/test.html'], {shell:true})
+
+waitOnProcess.on('close', () => {
+  console.log('starting cypress tests')
+  
+  const cypressProcess = spawn('npx', ['cypress', 'run', '--spec', 'cypress/integration/*'], {shell:true})
+  cypressProcess.stdout.on('data', data=>console.log(data.toString()))
+  cypressProcess.stderr.on('data', data=>console.error(data.toString()))
```

Comment:
I think there's a  way to call `spawn` such stdout/stderr are already piped to the correct places...  See here: https://nodejs.org/api/child_process.html#child_process_options_stdio

Reply:
thank you! Was wondering too!